update

```diff
@@ -61,12 +71,20 @@ def update_package_cache():
     print('    --> Collecting pkgbases...')
     # query all pkgbases per arch
     for arch in archs:
-        cmd = ['expac', '-S', '--config', get_configpath(arch), '%n %e']
+        cmd = ['expac', '-t', '%s', '-S', '--config', get_configpath(arch), '%n %e %v %b']
```

Comment:
Hmm why aren't we using pyalpm for this stuff?

Reply:
It has been added, so the next release we can start using it. https://github.com/archlinux/pyalpm/commit/f1513b49c3e2af9741fc1a65487b6413a4261de6